lib/data/pg-promise.js

```diff
@@ -63,16 +63,20 @@ function parseTable(table){
 function ensureTable(table, hardReset){
   var t = parseTable(table);
 
-  var ensureSchema = 'CREATE SCHEMA IF NOT EXISTS '+ t.schema+';';
-  var ensureTableQ = 'CREATE TABLE IF NOT EXISTS '+t.full+' ( data jsonb);';
-  var q = ensureSchema +' ' + ensureTableQ;
-
   if(hardReset){
     var drop = 'DROP TABLE IF EXISTS '+ t.full+'; ';
-    q = drop + q;
+    executeQuery(db, packageQuery(drop)).catch(function() {
```

Comment:
Split these multiple queries into single queries because I noticed only the first one was being executed locally. This was only happening if the Postgres schema and table didn't already exist (e.g. on a fresh install locally).